crates/bevy_ui/src/geometry.rs

```diff
@@ -211,21 +233,45 @@ impl Size {
     /// assert_eq!(size.width, Val::Px(100.0));
     /// assert_eq!(size.height, Val::Px(200.0));
     /// ```
-    pub fn new(width: Val, height: Val) -> Self {
+    pub const fn new(width: Val, height: Val) -> Self {
         Size { width, height }
     }
 
-    /// Creates a Size where both values are [`Val::Auto`].
+    /// Creates a new [`Size`] where both values are given in [`Val::Px`].
+    pub const fn px(width: f32, height: f32) -> Self {
```

Comment:
I'd actually really like the same for `UiRect`, it's annoyingly verbose to create now. But this can be on another PR.